packages/formatter-html/scripts/compile-locales.js

```diff
@@ -0,0 +1,46 @@
+const fs = require('fs');
+const path = require('path');
+const util = require('util');
+
+const globby = require('globby');
+const fsExtra = require('fs-extra');
+
+const writeFileAsync = util.promisify(fs.writeFile);
+const readFileAsync = util.promisify(fs.readFile);
+
+
+const compile = async () => {
```

Comment:
Probably worth adding a comment explaining *why* we're compiling the locales, specifically what the generated `assets/js/scan/_locales/*/messages.js` files get used for.